src/cpp/r/session/RConsoleActions.cpp

```diff
@@ -200,6 +200,10 @@ Error ConsoleActions::loadFromFile(const FilePath& filePath)
       if (error && !isFileNotFoundError(error))
          return error;
 
+      // bail if file is empty
+      if (contents.empty())
+         return Success();
```

Comment:
I was seeing periodic errors when restoring console actions; this is a small defensive fix I brought into this PR.